test/local_creds_test.go

```diff
@@ -167,13 +165,10 @@ func testLocalCredsE2EFail(dopts []grpc.DialOption) error {
 		EmptyCallF: func(context.Context, *testpb.Empty) (*testpb.Empty, error) {
 			return &testpb.Empty{}, nil
 		},
+		S: grpc.NewServer(grpc.Creds(local.NewCredentials())),
 	}
-
-	sopts := []grpc.ServerOption{grpc.Creds(local.NewCredentials())}
-	s := grpc.NewServer(sopts...)
-	defer s.Stop()
-
-	testgrpc.RegisterTestServiceServer(s, ss)
+	defer ss.S.Stop()
+	stubserver.StartTestService(nil, ss)
```

Comment:
why are you passing nil? You can pass `t` from caller and provide the same here?

Reply:
Done